File: Modul_5/Bahasa_Python/PRAK505-2510817120005-NorKhalisyah.py
def Biodata(tahunLahir, nama, asal):
    tahun_sekarang = 2020
    umur = tahun_sekarang - tahunLahir

    print(f"\nPerkenalkan Nama Saya: {nama}")
    print(f"Umur Saya: {umur} Tahun")
    print(f"Saya Adalah Angkatan: {tahun_sekarang}")
    print(f"Asal Saya dari: {asal}")

tahunLahir = int(input("Masukkan Tahun Lahir: "))
nama = input("Masukkan Nama: ")
asal = input("Masukkan Asal: ")

Biodata(tahunLahir, nama, asal)

File: Modul_5/Bahasa_C/PRAK505-2510817120005-NorKhalisyah.c
#include <stdio.h> 
 
void Biodata(int tahunLahir, char nama [20], char asal [15]){ 
int tahun_sekarang = 2020; 
int umur = tahun_sekarang - tahunLahir; 

printf("\nPerkenalkan Nama Saya: %s\n", nama);
printf("Umur Saya: %d Tahun\n", umur);
printf("Saya Adalah Angkatan: %d\n", tahun_sekarang);
printf("Asal Saya dari: %s\n", asal);

} 
 
int main() { 
int tahunLahir; 
char A[20], B[15]; 

printf("Masukkan Tahun Lahir: ");
scanf(" %d",&tahunLahir); 
printf("Masukkan Nama: ");
scanf(" %[^\n]%*c",A); 
printf("Masukkan Asal: ");
scanf(" %[^\n]%*c",B);
Biodata(tahunLahir, A, B); 
return 0; 
} 